src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -118,7 +118,8 @@ internal static bool CanGetSemanticInfo(CSharpSyntaxNode node, bool allowNamedAr
                         (node is ExpressionSyntax && (isSpeculative || allowNamedArgumentName || !SyntaxFacts.IsNamedArgumentName(node))) ||
                         (node is ConstructorInitializerSyntax) ||
                         (node is AttributeSyntax) ||
-                        (node is CrefSyntax);
+                        (node is CrefSyntax) ||
+                        (node is LocalFunctionStatementSyntax);
```

Comment:
this seems odd to me.  I wouldn't expect to be able to do things like GetSymbolInfo on local function.  I would expect to be able to do GetDeclaredSymbol